### ISSUE TITLE
Ability to edit password protected files without disabling protection

### ISSUE THREAD

**Issue by rgl1234** [ONLYOFFICE/DocumentServer#752]
Hello
We have some password-protected files that we would like to modify from onlyoffice/nextcloud. The files can be opened but unfortunately the passwort-protection is lost once the file is edited within onlyoffice.
Is there a plan to fix this problem?
René

**Comment by ShockwaveNN** [comment]
This issue is not related to Docker version of DocumentServer, moved to `DocumentServer` repo

**Comment by ShockwaveNN** [comment]
This is known issue (35279 in our private tracker), but currently we have no solution for this without compromising security
Please not, that editing password protected files is implemented in DesktopEditors

**Comment by rgl1234** [comment]
ok, I understand.... But then it would maybe be better to block saving those files from WebEditor. Because saving password-protected files without password is also a huge security-problem.....

**Comment by ShockwaveNN** [comment]
Currently we show warning about losing password:
![image](https://user-images.githubusercontent.com/668524/72718787-a7334400-3b87-11ea-8f96-42bd5630e069.png)
It's up to you to decide to decrypt file

**Comment by rgl1234** [comment]
ok...I understand...

**Comment by cpot** [comment]
is this serious ?

**Comment by ShockwaveNN** [comment]
> is this serious ?

If by this question you mean "How soon it will be implemented?" - I've just asked developer team and they answer that we are planning to add abilityo to store password in future version 6.2, but currently this work is not complete, so I cannot guarantee that it will be really released in 6.2.

**Comment by cpot** [comment]
We love onlyoffice, we try to push all our (large public) customers to drop MS-Office to Onlyoffice so they buy you licenses. 
However the roadmap is not the one we would like to achieve that.
On this particular issue (the other one is the drop-down management that customers waiting for > 2 years), they just stay on MS-Office because the functionality does not exist, worse they see it as a major flaw (loosing password protected file)

As a software vendor we know it is hard to follow the right roadmap with the correct priorities.

As a product is becoming more popular, challenge increases... Becoming more popular is a good news, the downside is that they must not be disappointed, we have the same challenge. 

Perhaps introducing a functionality voting system could be a good idea even if at some point you must have also your own roadmap too

**Comment by ShockwaveNN** [comment]
@cpot Thank you for understanding, and I'll send your words to our management. 

Some time ago we got voting system, but it was shut down for reason unknown to me

**Comment by cpot** [comment]
@ShockwaveNN some time it can create frustration among users/customers if roadmap is different from voting choices.

**Comment by cpot** [comment]
Any update on this important issue ?

**Comment by Rita-Bubnova** [comment]
@cpot, I checked this bug and I confirm - it was fixed in ONLYOFFICE Document Server version 7.0.1.
Sorry, we forgot to mention that issue was fixed.

**Comment by ShockwaveNN** [comment]
I'll close this issue but feel free to comment or create new issues if you got any troubles with solution of this issue
